users/management/commands/unblock_users_test.py

```diff
@@ -0,0 +1,90 @@
+"""retire user test"""
```

Comment:
I believe the tests should not be placed directly in the commands folder as it is very specific to management commands, I think we should create a separate folder under management named as tests. Any thoughts?